librosa/core/audio.py

```diff
@@ -425,7 +425,7 @@ def to_mono(y):
 
     Parameters
     ----------
-    y : np.ndarray [shape=(2,n) or shape=(n,)]
+    y : np.ndarray [shape=(k,n) or shape=(n,)]
```

Comment:
this is what I would think the possible shapes of raw audio time-series could be.

Reply:
good catch, I forgot to fix that one. :grin: